packages/mco/components/modals/app-manage-policies/helper/messages.tsx

```diff
@@ -0,0 +1,77 @@
+import * as React from 'react';
+import { useCustomTranslation } from '@odf/shared/useCustomTranslationHook';
+import { TFunction } from 'i18next';
+import {
+  Alert,
+  AlertVariant,
+  Button,
+  ButtonVariant,
+} from '@patternfly/react-core';
+import { ModalActionContext, PolicyListViewState } from '../utils/reducer';
+
+const AlertMessage: React.FC<AlertMessageType> = (props) => {
+  const { title, variant, children, t } = props;
```

Comment:
nits: (since I have seen these types of comments of other PRs)
```suggestion
const AlertMessage: React.FC<AlertMessageType> = ({ title, variant, children}) => {
```